code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -369,8 +366,6 @@
 				harvest(user)
 			return
 
-	..()

```

Comment:
this is very much needed, you won't be able to attack animals with items if you remove that.